### PR TITLE
Take some resources from Mysos task to assign to the executor.

### PR DESCRIPTION
**Pull request by xujyan**
For #54 

When executors don't have resources assigned, Mesos slave kills the container right after the task's terminal update is received.

### CODE REVIEW

File: mysos/scheduler/scheduler.py [hunk 1]
```diff
@@ -210,6 +212,12 @@ def create_cluster(
         raise ValueError("Invalid number of cluster nodes: %s" % num_nodes)
 
       resources = parse_size(size)
+
+      if (resources['cpus'] <= EXECUTOR_CPUS_EPSILON or
+          resources['mem'] <= EXECUTOR_MEM_EPSILON or
+          resources['disk'] <= EXECUTOR_DISK_EPSILON):
+        raise ValueError("Instance 'size' too small")
```
Comment (vinodkone) [inline mysos/scheduler/scheduler.py]: print the minimum sizes required?

**Comment by vinodkone** [comment]
:+1:

File: mysos/scheduler/scheduler.py (hunk 1 above)
Comment (xujyan) [inline mysos/scheduler/scheduler.py]: :ok_hand:

**Pull request merged**